subworkflows/nf-core/bam_variant_calling_somatic_mutect2/main.nf

```diff
@@ -0,0 +1,196 @@
+//
+// Run GATK mutect2 in tumor normal mode, getepileupsummaries, calculatecontamination, learnreadorientationmodel and filtermutectcalls
+//
+
+include { GATK4_CALCULATECONTAMINATION                                      } from '../../../modules/nf-core/gatk4/calculatecontamination/main'
```

Comment:
accidentally added?

Reply:
I don't know what you're talking about ;-)